notebooks/GP-Marginal.ipynb

```diff
@@ -0,0 +1,381 @@
+{
```

Comment:
<ul><li><s>#Note that we have to call </s><code><s>eval()</s></code><s> because PyMC3 built on top of Theano</s><span class="ql-cursor">﻿</span></li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc4/pull/309/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>